gemet/thesaurus/management/commands/check_spreadsheet.py

```diff
@@ -0,0 +1,117 @@
+from optparse import make_option
+
+from django.core.management import CommandError
+from django.core.management.base import BaseCommand
+from gemet.thesaurus.models import Property
+
+import re
+import xlrd
+
+LABELS = {
+    'RT (GEMET)': True,
+    'RT (new)': False,
+    'BT (GEMET)': True,
+    'BT (new)': False,
+    'NT (GEMET)': True,
+    'NT (new)': False,
+}
+
+
+def check_new_term_existence(new_terms, term):
+    return term in new_terms
+
+
+def check_term_database_existence(term):
+    return Property.objects.filter(value=term).exists()
+
+
+def split_text_into_terms(raw_text):
+
+    raw_text = raw_text.strip()
+    pattern = re.compile("[^a-zA-Z\d\s\-'\)''\(':]")
+    term_list = pattern.split(raw_text)
+    correct_term_list = []
+    for term in term_list:
+        correct_terms = map(lambda term: term.strip(' \t\n;,.:').lower(),
+                            term.split('\n'))
+        correct_terms = filter(lambda term: term != '', correct_terms)
+        correct_term_list.extend(correct_terms)
+    return correct_term_list
+
+
+def row_idx_to_name(idx):
+    result = ""
+    while True:
+        if idx > 26:
+            idx, r = divmod(idx - 1, 26)
+            result = chr(r + ord('A')) + result
+        else:
+            return chr(idx + ord('A') - 1) + result
+
+
+def to_excel_row_column(row_idx, column_idx):
+    row = row_idx_to_name(row_idx + 1)
+    return row + str(column_idx + 2)
+
+
+def check_term_existence(term, term_type, excel_cell_value, new_terms):
+    if term_type is not None:
+        message = 'Error at cell {}: "{}"'.format(excel_cell_value, term)
+        if term_type:
+            if not check_term_database_existence(term):
+                message += ' not defined in database.'
+                message += ' Term found in column "Label".' \
+                    if check_new_term_existence(new_terms, term) else ''
+                print message
+        else:
+            if not check_new_term_existence(new_terms, term):
+                message += ' not found in column "Label".'
+                message += ' Term found in database.' \
+                    if check_term_database_existence(term) else ''
+                print message
+
+
+class Command(BaseCommand):
+    help = 'Check if spreadsheet terms are consistent.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            "-f",
+            "--file",
+            dest="filename",
+            help="specify excel import file",
+            metavar="FILE"
+        ),
+    )
```

Comment:
Use `add_arguments()` method for positional arguments ([docs](https://docs.djangoproject.com/en/1.10/howto/custom-management-commands/#accepting-optional-arguments)).